.rubocop.yml

```diff
@@ -7,7 +7,7 @@ FrozenStringLiteralComment:
   Enabled: false
 
 LineLength:
-  Max: 99
+  Max: 110
```

Comment:
Let's try to stick with the 99 limit. You can always split your code into more than one line if you need to.